docs/schema/animated-properties/keyframe.json

```diff
@@ -7,17 +7,17 @@
         {
             "properties": {
                 "t": {
-                    "title": "Time",
+                    "title": "Keyframe Time",
```

Comment:
Not sure I agree with these, it's describing the keyframe, so why repeat it in the title of the properties?

Reply:
Because it's a generic name I wanted to create a unique title.